src/Helper/Initializer/MerchantApi.php

```diff
@@ -0,0 +1,54 @@
+<?php
+/**
+ * Copyright Shopgate Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * @author    Shopgate Inc, 804 Congress Ave, Austin, Texas 78701 <interfaces@shopgate.com>
+ * @copyright Shopgate Inc
+ * @license   http://www.apache.org/licenses/LICENSE-2.0 Apache License, Version 2.0
+ */
+
+namespace Shopgate\Base\Helper\Initializer;
+
+use Shopgate\Base\Model\Config as MainConfig;
+use ShopgateBuilder;
+use ShopgateMerchantApi;
+
+class MerchantApi
+{
+    /** @var MainConfig */
+    private $config;
+
+    /**
+     * @param MainConfig $config
+     */
+    public function __construct(
+        MainConfig $config
```

Comment:
can set to one line as not too many DI's are here (makes konstantin's of the world happy)